model/retry_test.go

```diff
@@ -94,7 +101,7 @@ func TestRetryStructLevelValidation(t *testing.T) {
 				Increment: "PT5S",
 				Jitter:    floatstr.FromString("P5S"),
 			},
-			err: `Key: 'Retry.Jitter' Error:Field validation for 'Jitter' failed on the 'reqiso8601duration' tag`,
+			err: `Key: 'Retry.Jitter' Error:Field validation for 'Jitter' failed on the 'iso8601duration' tag`,
```

Comment:
this message is coming from the validateFunc, right?
Do you think we can make this message easier to understand?

Something like "field is not iso8601duration"

Reply:
Yes, we should custom the error message to have more description, and this should be done later. For how to implement custom error message, maybe https://github.com/go-playground/validator/issues/559 will helpful.